mphys/core/multipoint.py

```diff
@@ -46,6 +46,31 @@ def mphys_add_scenario(self, name, scenario, coupling_nonlinear_solver=None,
         self.mphys_coupling_solvers.append((scenario, solver_tuple))
         return self.add_subsystem(name, scenario)
 
+    def mphys_connect_scenario_coordinate_source(self, source, scenarios, disciplines):
```

Comment:
I had left this out when refactoring the naming convention because I think this form would only work if the source is the mesh subsystem. If you have a geometry component as the source of the coordinates, then I think this would require some more sophisticated logic.

`src = discipline_vars.Mesh.COORDINATES` for the mesh subsubsystem but
`src = discipline_vars.Geometry.COORDINATES_INPUT` if it comes from a geometry component.

I could not figure out how to generalize this, so I left it to the user to explicitly do the connections themselves. If you know how we could do this for the geometry and the mesh component sources, then we can put this method back in.

Reply:
Should be removed  now